app/pages/sections/catalogue-solutions/select-price/contextCreator.test.js

```diff
@@ -2,6 +2,75 @@ import manifest from './manifest.json';
 import { getContext } from './contextCreator';
 import { baseUrl } from '../../../../config';
 
+const solutionPricingData = {
+  id: 'supp-1',
+  name: 'name',
+  prices: [
+    {
+      type: 'flat',
+      currencyCode: 'GBP', // ISO Currency Code
+      itemUnit: {
+        name: 'patient',
+        description: 'per patient',
+      },
+      timeUnit: {
+        name: 'year',
+        description: 'per year',
+      },
+      price: 1.64,
+    },
+    {
+      type: 'flat',
+      currencyCode: 'GBP', // ISO Currency Code
+      itemUnit: {
+        name: 'licence',
+        description: 'per licence',
+      },
+      price: 525.052,
+    },
+    {
+      type: 'tiered',
+      currencyCode: 'GBP', // ISO Currency Code
+      itemUnit: {
+        name: 'consultation',
+        description: 'per consultation',
+        tierName: 'consultations',
+      },
+      timeUnit: {
+        name: 'month',
+        description: 'per month',
+      },
+      tieringPeriod: 3,
+      tiers: [
+        {
+          start: 1,
+          end: 10,
+          price: 700.0,
+        },
+        {
+          start: 11,
+          price: 400.0,
+        },
+      ],
+    },
+  ],
+};
+
+const returnedPriceArray = [{
+  id: 'selectSolutionPrice',
+  mainAdvice: 'Select list price',
+  options: [{
+    text: '£1.64 per patient per year',
+    value: '£1.64 per patient per year',
+  }, {
+    text: '£525.052 per licence ',
+    value: '£525.052 per licence ',
+  }, {
+    html: '<div>1 - 10 consultations £700 per consultation per month</div><div>11+ consultations £400 per consultation per month</div>',
+    value: '<div>1 - 10 consultations £700 per consultation per month</div><div>11+ consultations £400 per consultation per month</div>',
```

Comment:
mmm this is interesting.. I'm not sure if we want this to be the value. When they select a price do we want to somehow uniquely identify this price later on so we can show it. I'll chat the the API guys and see if I can bit more info about this

Reply:
was going to change this when we get the details of the put as yeah not an ideal value to be sending back

Reply:
I have just spoke to @sam-roberts-bjss about this and we feel like the API should be returning an ID that we can use to identify the price selected. The API will have to make a change on their side to reflect this. But for now can we assume that the JSON we will get back from the `Pricing` endpoint will return this ID per price. So it will look something like this
```
{
    "id": "<ID>",
    "name": "name",
    "prices": [
      {
        **"priceId": "0001"**,
        ...
      },
      {
        **"priceId": "0002"**,
       ...
      },
   ],
}
```
Use this priceId as the value for each radioButton